.github/renovate.json5

```
@@ -3,6 +3,7 @@
     "config:base",
```

I think we can enable the automerge for minor.

```suggestion
    "config:base",
    ":automergeMinor",
    ":label(renovate/{{depName}})"
```